nixos/doc/manual/development/settings-options.section.md

```diff
@@ -364,6 +364,67 @@ have a predefined type and string generator already declared under
     and returning a set with [CDN](https://github.com/dzikoysk/cdn)-specific
     attributes `type` and `generate` as specified [below](#pkgs-formats-result).
 
+`pkgs.formats.kdl` { }
+
+:   A function taking an empty attribute set (for future extensibility)
```

Comment:
Hello. It is the future. There should be an option to choose which version of KDL to output.

There are exactly two versions, KDL v1.0.0 and KDL v2.0.0. It would be nice if we could make this *explicitly specified* and *required* before this format is merged.